libasn1print/asn1printproto.c

```diff
@@ -2186,6 +2223,183 @@ asn1extract_columns(asn1p_expr_t *expr, proto_module_t *proto_module, char *mod_
 	return 0;
 }
 
+// ToDo - this function would be reworked later to handle VALUE SET better
```

Comment:
lots of comments - do you intend coming back to it? 
I would remove the commented out section and retvieve it from git history unless maybe you're going to use it in your next patch

Reply:
Intent is to use it in next patch. It has lots of other valued comments for future work..
Can I proceed and merge, @SeanCondon?

Reply:
yep - go for it